### DISCUSSION TITLE
Idea for a new settings page layout

### DISCUSSION THREAD

**Discussion by DNin01** [ScratchAddons/ScratchAddons#5864] [Ideas]
![A mock-up of a settings page redesign. At the left, there's a sidebar with category icons and the Scratch Addons logo. In the middle, there's a list of addons divided into groups. One of them is highlighted. On the right, there's a details pane displaying the description and settings of the highlighted addon.](https://user-images.githubusercontent.com/106490990/229261373-bd2ebdf8-f45d-4f46-8210-c9b0735e6656.png)
Created with Scratch's costume editor. Star shape reused from a library sprite. Also, we don't have to go with blue as the accent color.

We could also:
- Display an image above the addon's name on the details pane

This isn't a complete mock-up, it's like a blueprint.

**Comment by Platformer444** [comment]
Star shaped-button for favourite addons?

**Reply by DNin01** [reply]
Recommended addons.

**Reply by Platformer444** [reply]
Oh okay

**Comment by Secret-chest** [comment]
Very good. It's much more useable than the `v2-design`.

**Reply by WorldLanguages** [reply]
Has this "two column design" been suggested ever before? I think it's worth considering

**Reply by Secret-chest** [reply]
> Has this "two column design" been suggested ever before? I think it's worth considering

I'm going to draw some more professional mockups (Inkscape) in 2 days

**Reply by Secret-chest** [reply]
Starting work now.

**Reply by Secret-chest** [reply]
Done! #5893
